.gitmodules

```diff
@@ -21,3 +21,6 @@
 [submodule "rts/lib/tracy"]
 	path = rts/lib/tracy
 	url = https://github.com/wolfpld/tracy.git
+[submodule "vclibs64"]
```

Comment:
Please don't reference this as submodule. Those files are not needed by anyone that isn't using visual studio so not everyone should be forced to download them.

For example if you reference those, you should definitely also reference linux static libs and mingw libs as those are used as for actual production builds.

Reply:
There is alternative approach that could be applied to still automate it, in CMake, when we detect MSVC, get the vclibs64 using https://cmake.org/cmake/help/latest/module/FetchContent.html

Then it will only impact visual studio users and make it convenient for them not forcing mingw+linux users to fetch this repo.

Reply:
Sounds cleaner though worth mention it's we're talking 58.8 MB out of 18.8 GB

Reply:
Current clean checkout is exactly 412MiB, not 18GiB.

Reply:
I guess mine was so big bc I had multiple build-msvc folders